modules/mobile/app/controllers/mobile/messaging_controller.rb

```diff
@@ -20,6 +20,8 @@ def authorize
     end
 
     def raise_access_denied
+      rails.logger.info('SM ACCESS DENIED', current_user.mhv_account_type, current_user.mhv_correlation_id,
```

Comment:
Are you sure this works? Should rails be capitalized?

Reply:
Whoops, good catch

Reply:
Interesting that rubymine seemed to interpret it correctly either way. 🤷